README.md

```diff
@@ -12,9 +12,15 @@ Used as a camera simulator if you have pre-recorded video you want to stream, or
 This is a work in progress.
 
 ## Install
-`colcon build --symlink-install`
+``` bash
+# Install natsort (is not in rosdep)
+apt update -y
+apt install -y python3-natsort
+rosdep install -i --from-path . -y
```

Comment:
Maybe you could add a sub-header here to state that this is for Linux / ubuntu systems ? 

edit: sorry missed below

Reply:
It makes sense, in last commit and I've made it clearer